src/modules/todos/controllers.js

```diff
@@ -1,144 +1,154 @@
-import { todos} from '../database/database.js';
-import AppException from '../../utils/exceptions/AppException.js';
-import { v4 } from 'uuid';
+import { v4 } from "uuid";
 
+import pool from "../../database/database.js";
+import AppException from "../../utils/exceptions/AppException.js";
+import queries from "./queries.js";
 
-export function getTodos(req, res) {
-	res.json(todos.filter((todo) => todo.userId === req.userId));
-};
+export async function getTodos(req, res, next) {
+	try {
+		const allTodos = await pool.query(queries.allTodos, [req.userId]);
+		res.json(allTodos.rows);
+	} catch (err) {
+		next(err);
+	}
+}
 
-export function createTodo(req, res, next) {
+export async function createTodo(req, res, next) {
 	try {
 		let today = new Date();
-		
-		let strDate = 'Y-m-d h:M:s'
-			.replace('Y', today.getFullYear())
-			.replace('m', today.getMonth()+1)
-			.replace('d', today.getDate())
-			.replace('h', today.getHours())
-			.replace('M', today.getMinutes())
-			.replace('s', today.getSeconds());
-		const id = v4();
-		const userId = req.userId;
-		const title = req.body.title;
-		const status = req.body.status;
-		const dueDate = req.body.dueDate;
+
+		let strDate = "Y-m-d h:M:s"
```

Comment:
This would no longer be needed since the the columns will be of the timestamp data type.